lib/schema-loaders/local-loader.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const YAML = require('js-yaml');
+const chokidar = require('chokidar');
+const EventEmitter = require('events');
+const logger = require('lllog')();
+
+const OpenAPISchemaNotFound = require('../errors/openapi-schema-not-found-error');
+const OpenAPISchemaMalformed = require('../errors/openapi-schema-malformed-error');
+
+module.exports = class LocalSchemaLoader extends EventEmitter {
+
+	constructor(schemaPath) {
+		super();
+		this.schemaPath = schemaPath.match(/^\//) ? schemaPath : path.join(process.cwd(), schemaPath);
```

Comment:
Correct me if I am wrong but the ternary statement here is checking to see if the path provided is an absolute path right? In which case it takes the path as is, otherwise it assumed it is relative and joins the path with the current working directory.

This regex will only work on *nix based systems and won't work on Windows.
Node has the `path.isAbsolute(dir)` [method](https://nodejs.org/api/path.html#path_path_isabsolute_path) available which should give a platform independent solution to that problem.

```suggestion
		this.schemaPath = path.isAbsolute(schemaPath) ? schemaPath : path.join(process.cwd(), schemaPath);
```

Reply:
Yeah, you're right. Looks like sometimes I forget that windows even exists.. 😂

Reply:
Ha. It's very easy to do.
It wasn't until I looked again that I realised that it was already like that in the CLI and the code was just ported over.
I guess it's not really been an issue since I am guessing that most people are running this under Docker or with relative paths.